homeassistant/components/husqvarna_automower_ble/lawn_mower.py

```diff
@@ -0,0 +1,156 @@
+"""The Husqvarna Autoconnect Bluetooth lawn mower platform."""
+
+from __future__ import annotations
+
+import logging
+
+from homeassistant.components import bluetooth
+from homeassistant.components.lawn_mower import (
+    LawnMowerActivity,
+    LawnMowerEntity,
+    LawnMowerEntityFeature,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import HusqvarnaAutomowerBleEntity, HusqvarnaCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up AutomowerLawnMower integration from a config entry."""
+    coordinator: HusqvarnaCoordinator = hass.data[DOMAIN][config_entry.entry_id]
+    model = coordinator.model
+    address = coordinator.address
+
+    async_add_entities(
+        [
+            AutomowerLawnMower(
+                coordinator,
+                "automower" + str(model) + "_" + str(address),
+                model,
+                LawnMowerEntityFeature.PAUSE
+                | LawnMowerEntityFeature.START_MOWING
+                | LawnMowerEntityFeature.DOCK,
+            ),
+        ]
+    )
+
+
+class AutomowerLawnMower(HusqvarnaAutomowerBleEntity, LawnMowerEntity):
+    """Husqvarna Automower."""
+
+    def __init__(
+        self,
+        coordinator: HusqvarnaCoordinator,
+        unique_id: str,
+        name: str,
+        features: LawnMowerEntityFeature = LawnMowerEntityFeature(0),
+    ) -> None:
+        """Initialize the lawn mower."""
+        super().__init__(coordinator)
+        self._attr_name = name
```

Comment:
The lawn mower is the main feature of the device, so the name of the lawn mower should be set to `_attr_name = None`. This way, the name of this entity is decided by the name of the device, which is consistent